docs/src/pages/guides/FHIRClinicalReasoningGuide.md

```diff
@@ -0,0 +1,41 @@
+# Overview
+
+(BETA) Starting in IBM FHIR 4.9.0 initial support was added for the [Clinical Quality Language (CQL)](https://cql.hl7.org/) and FHIR operations defined in the [Clinical Practice Guidelines (CPG) ](https://build.fhir.org/ig/HL7/cqf-recommendations/) and [Quality Measure (cqf-measures)](https://build.fhir.org/ig/HL7/cqf-measures/index.html) Implementation Guides (IG) that leverage CQL. These IGs build upon the resources and operations defined in the [FHIR Clinical Reasoning Module](http://www.hl7.org/FHIR/clinicalreasoning-module.html). 
```

Comment:
General terminology we have is `experimental`, it may be more appropriate. 

Also the links are the build site, maybe it should be to the specific released version `http://hl7.org/fhir/us/cqfmeasures/stu2/` or mention the commit that this is built at or the day at which it was last relevant.

Reply:
```suggestion
(BETA) Starting in IBM FHIR Server 4.9.0 initial support was added for the [Clinical Quality Language (CQL)](https://cql.hl7.org/) and FHIR operations defined in the [Clinical Practice Guidelines (CPG) ](https://build.fhir.org/ig/HL7/cqf-recommendations/) and [Quality Measure (cqf-measures)](https://build.fhir.org/ig/HL7/cqf-measures/index.html) Implementation Guides (IG) that leverage CQL. These IGs build upon the resources and operations defined in the [FHIR Clinical Reasoning Module](http://www.hl7.org/FHIR/clinicalreasoning-module.html). 
```

Reply:
We're approved to use IBM FHIR Server